src/snyk/base/services/authenticationService.ts

```diff
@@ -81,6 +82,11 @@ export class AuthenticationService implements IAuthenticationService {
     }
   }
 
+  async initiateLogout(): Promise<void> {
+    await this.configuration.clearToken();
+    await this.contextService.setContext(SNYK_CONTEXT.LOGGEDIN, false);
```

Comment:
We kinda should handle all running scans (e.g. by stopping them), but there's no existing code for it. I think it's fine to have it as-is for now but we should keep that in mind :)

Reply:
ah that is very good point.